openedx/core/djangoapps/password_policy/apps.py

```diff
@@ -0,0 +1,27 @@
+"""
+Configuration for password_policy Django app
+"""
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+from openedx.core.djangoapps.plugins.constants import ProjectType, SettingsType, PluginURLs, PluginSettings
+
+
+class PasswordPolicyConfig(AppConfig):
+    """
+    Configuration class for bookmarks Django app
```

Comment:
Copy and pasted.

Reply:
Is this class going to be around for a while (like, is this going to be removed after rollout)?

If it's staying, do you envision the password validators moving here? The "password policy" name seems like they would belong here.